### ISSUE TITLE
crash of application due to `nodeIntegration` statement

### ISSUE THREAD

**Issue by Gbkng** [fireship-io/223-electron-screen-recorder#52]
# Environment

I've tested commit feb08cd (present HEAD of master), on Ubuntu 22.04 with npm v18.7.1 (Latest LTS: Hydrogen). The app  crashes after few seconds when I try to launch it with `npm run start`.

# Log

Snapshot of DevTools just after the crash:

![image](https://github.com/fireship-io/223-electron-screen-recorder/assets/68586594/0331518a-ba61-4a0c-a189-64723cf75b9b)

Log message:

```sh
$>npm run start

> my-app@1.0.0 start
> electron-forge start

✔ Checking your system
✔ Locating application
✔ Loading configuration
✔ Preparing native dependencies [0.2s]
✔ Running generateAssets hook

[234469:0815/201340.050509:ERROR:CONSOLE(1)] "Uncaught (in promise) TypeError: Failed to fetch", source: devtools://devtools/bundled/panels/elements/elements.js (1)
```

# Possible solution

It seems that the line causing the error is `src/index.js:15` : 

```js
const createWindow = () => {
  // Create the browser window.
  const mainWindow = new BrowserWindow({
    width: 800,
    height: 600,
    webPreferences: {
      nodeIntegration: true // ERROR HERE
    }
  });
```

When I comment this line, the application does not crash anymore. Moreover, it seems broadly realated to a comment I've found below the related [youtube video](https://www.youtube.com/watch?v=3yqDxhR2XxE&t=150s) : 

> @andyholm-jensen9075
> As of Electron v9 you can't call remote on the renderer unless you set the enableRemoteModule as true on the BrowserWindow (like you did with nodeIntegration). Great tutorial nonetheless. :)

In conclusion, it seems that the error is due to difference of version of `electron` version between my environment and the video tutorial. **Commenting the aforementioned line seems to solve the issue.**
